### PR TITLE
#1132: Fix bug in SlowInputStream

### PR DESCRIPTION
**Pull request by andreoss**
Per #1132: 
- Add test
- Fix bug

### CODE REVIEW

**Comment by victornoel** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/yegor256/cactoos/issues/1467#issuecomment-695809382))

@victornoel This is what I know about this job in C63314D6Z, as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#1467](https://github.com/yegor256/cactoos/issues/1467) is [in scope](http://www.zerocracy.com/policy.html#14) for 2days
  * The role is `REV`
  * The job is assigned to @fabriciofx[/z](https://www.0crat.com/u/fabriciofx) for [2days](http://www.zerocracy.com/policy.html#8)
  * There is no monetary reward attached, it's a [free](http://www.zerocracy.com/policy.html#2) job
  * The job doesn't have any [impediments](http://www.zerocracy.com/policy.html#9)
  * The [budget](http://www.zerocracy.com/policy.html#4) is 15 minutes/points
  * These users are banned and won't be assigned:
    * @andreoss[/z](https://www.0crat.com/u/andreoss): This user reported the ticket
  * Job [footprint](https://www.0crat.com/footprint/C63314D6Z?q=%7Bjob%3A%27gh%3Ayegor256%2Fcactoos%231467%27%2Ctype%3A%7B%24not%3A%2F%5ENotify%2F%7D%7D) (restricted area)

File: src/test/java/org/cactoos/io/SlowInputStreamTest.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+/**
+ * Test for {@link SlowInputStream}.
+ *
+ * @since 0.47
+ * @checkstyle MagicNumberCheck (20 lines)
+ */
+final class SlowInputStreamTest {
+
+    @Test
+    void readsSigned() throws Exception {
```
Comment (fabriciofx) [inline src/test/java/org/cactoos/io/SlowInputStreamTest.java:39]: @andreoss please, add one more test to assure the class `SlowInputStream` can convert a unsigned byte to int properly.

**Review by fabriciofx — CHANGES_REQUESTED** [review]
@andreoss can you take a look, please?

File: src/test/java/org/cactoos/io/SlowInputStreamTest.java (hunk 1 above)
Comment (andreoss) [reply src/test/java/org/cactoos/io/SlowInputStreamTest.java:39]: @fabriciofx Done.

**Review by andreoss — COMMENTED** [review]
@fabriciofx Please, take a look again

**Review by fabriciofx — APPROVED** [review]
@andreoss It seems fine to me. @victornoel can you merge it, please?

**Comment by victornoel** [comment]
@fabriciofx thx

@andreoss the build is now failing unfortunately :)

**Comment by andreoss** [comment]
@victornoel Fixed

**Comment by victornoel** [comment]
@rultor merge

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @fabriciofx[/z](https://www.0crat.com/u/fabriciofx), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by sereshqua** [comment]
@fabriciofx please make sure you will find at least 3 issues during next CR, thanks

**Comment by sereshqua** [comment]
@0crat quality acceptable

**Pull request merged**